adoc/admin-migrate-caasp4.5.adoc

```diff
@@ -0,0 +1,74 @@
+== Migration to {productname} 4.5
+
+1) The OS must be connected to SCC. Otherwise the migration will not work.
+In order to reconnect your OS to SCC, you run:
+
+----
+SUSEConnect -r <your SCC key> SUSEConnect -p sle-module-containers/15.1/x86_64 -r <your SCC key>
+----
+
+2) You also need the new `zypper migration` plugin. This plugin is used to perform the needed tasks in order to migrate the node itself to the latest version of {platform}, such as updating the repositories to the new ones, and calling `zypper dup`. This plugin is provided by the `zypper-migration-plugin` package. Therefore, install the `zypper-migration-plugin` package:
+
+----
+zypper -n in zyppper-migration-plugin
+----
+
+3) Then, run the newly installed `zypper-migration` plugin (on the management node first, then on the rest of the nodes):
+
+----
+zypper migration --non-interactive --auto-agree-with-licenses
+----
+
+4) Check if `skuba` was indeed upgraded for 4.5:
+
+----
+skuba version
+----
+
+[IMPORTANT]
+====
+The version must be >= `skuba-2.1`. `skuba 2` corresponds to {productname} 4.5, while `skuba 1.0-1.4` corresponds to {productname} 4.
+====
+
+5) And now run the skuba cluster upgrade commands as it's done below.
+
+- First, check if there are any addons or components to upgrade before you upgrade the nodes:
+----
+skuba cluster upgrade plan
+skuba addon upgrade plan
+skuba addon upgrade apply
+----
+
+- Then, check with `cluster status` if all nodes have the same kubernetes version:
```

Comment:
... `which must be 1.17.x`

Reply:
I thought 4.5.0 contains k8s 1.18?

Reply:
> I thought 4.5.0 contains k8s 1.18?

That's before the upgrade takes place. The actual command that upgrades kubernetes to 1.18 is "skuba node upgrade apply --user sles --sudo --target IP"

Reply:
> ... `which must be 1.17.x`

Will add that right now...